content/metadata.yaml

```
@@ -53,3 +53,17 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-23
+    -
```

```suggestion
  -
```
I think the spacing might matter